movies-manager/src/containers/main/main.tsx

```diff
@@ -161,35 +148,65 @@ export function Main(props: IMainProps): JSX.Element {
             </li>;
         });
 
-    return <main className={blockName}>
+    return loadConfig.isLoading ? <LoadingIndicator /> :
+        !loadConfig.isLoaded && <ErrorHandler errorMessage={errorInfo.message}/>
```

Comment:
Did you try to throw an Error in one of your thunks, do not think this will work

Reply:
Done